tests/e2e/bindings/bindings_test.go

```diff
@@ -120,6 +130,25 @@ func TestBindings(t *testing.T) {
 	require.Equal(t, testMessages[1:], decodedResponse.ReceivedMessages)
 	// one message fails.
 	require.Equal(t, testMessages[0], decodedResponse.FailedMessage)
+
+	// act for http
```

Comment:
nitpick act for gRPC
```suggestion
	// act for gRPC
```